qml/Ping360Status.qml

```diff
@@ -10,12 +10,20 @@ Item {
     property var ping: null
 
     Rectangle {
-        anchors.centerIn: parent
-
+        id: rect
```

Comment:
It would be nice to have x and y equal to `x: (root.width - rect.width)/2 and y: 0`, right now the rect starts in the top left corner.

Reply:
![image](https://user-images.githubusercontent.com/1215497/62565062-62775300-b85c-11e9-9503-1e6c3ee96ffc.png)

Reply:
I'll do. I tried to do something similar but the root triggered a change in width during the drag while the drawing was updating and that made the rectangle jump.